bazel/foreign_cc/luajit.patch

```diff
@@ -65,11 +43,31 @@ index f56465d..3f4f2fa 100644
 
  ##############################################################################
 EOF
-
-patch -p1 < luajit_make.diff
-
-# Default MACOSX_DEPLOYMENT_TARGET is 10.4, which will fail the build at link time on macOS 10.14:
-# ld: library not found for -lgcc_s.10.4
-# This doesn't affect other platforms
-MACOSX_DEPLOYMENT_TARGET=10.6 DEFAULT_CC=${CC} TARGET_CFLAGS=${CFLAGS} TARGET_LDFLAGS=${CFLAGS} \
-  CFLAGS="" make V=1 PREFIX="$THIRDPARTY_BUILD" install
+diff --git a/build.py b/build.py
+new file mode 100755
+index 0000000..b069ef6
+--- /dev/null
++++ b/build.py
+@@ -0,0 +1,21 @@
++#!/usr/bin/env python
++
++import argparse
++import os
++
++def main():
++    parser = argparse.ArgumentParser()
++    parser.add_argument("--prefix")
++    args = parser.parse_args()
++    os.chdir(os.path.dirname(os.path.realpath(__file__)))
++
++    os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.6"
++    os.environ["DEFAULT_CC"] = os.environ.get("CC", "")
++    os.environ["TARGET_CFLAGS"] = os.environ.get("CFLAGS", "")
++    os.environ["TARGET_LDFLAGS"] = os.environ.get("CFLAGS", "")
++    os.environ["CFLAGS"] = ""
++    os.environ["LSAN_OPTIONS"] = "exitcode=0"
++
++    os.system('make V=1 PREFIX="{}" install'.format(args.prefix))
```

Comment:
Arguably we could generate a `Makefile` here that wraps the existing one, allowing `configure_make` to follow its natural two step process. I don't know if this would be anything other than "make the square peg fit the circle hole" improvement, but it's probably not very hard to do and might make any debugging or logging for `configure_make` better aligned.

Reply:
The difficult part of that is the prefix argument passed to configure script so we would have both script and wrapper Makefile, sounds like more hack :P